content/Community/committee.en.md

```diff
@@ -5,6 +5,10 @@ type: "post"
 author: "gaelcolas"
 ---
 
+Here's an explanation of what the DSC Committee is attempting to help with, and how we're trying to help the community.
+
+[Come and chat to us](http://slack.poshcode.org) if you'd like to discuss how we can improve!
```

Comment:
We should reference the new Join the conversation page here instead (in PR currently).